src/E2_economics.py

```diff
@@ -3,6 +3,7 @@
 from src.constants_json_strings import (
```

Comment:
git META tip: this could be a good use of `git commit --amend` (because you have 4 commits with very similar commit message one after the other)

Reply:
Yeaaaaahhh... I know there was something to spare you all the mails. Sorry about that. I am never using the terminal for git, but always the gui, so I foget about that stuff...

Reply:
I use both, gui for commits, then terminal for amend and rebase :)